### DISCUSSION TITLE
Prism Container

### DISCUSSION THREAD

**Discussion by aliahdev** [PrismLibrary/Prism#2869] [.NET MAUI]
Can't we have ContainerExtensions using MAUI DependencyInjection instead of DryIoC?

**Comment by dansiegel** [comment]
Your question doesn't make sense. If you're asking about Microsoft.Extensions.DependencyInjection which is the default DI container for MAUI, then the answer is no. Microsoft.Extensions.DependencyInjection is an immutable container which is not compatible with Prism.
